<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>nacelleTaskPanel</class>
 <widget class="QDialog" name="nacelleTaskPanel">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>343</width>
    <height>293</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Nacelle parameters</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <layout class="QGridLayout" name="gridLayout">
       <item row="15" column="1">
        <widget class="QDoubleSpinBox" name="sbRevolveAngle">
         <property name="suffix">
          <string>Â°</string>
         </property>
         <property name="decimals">
          <number>1</number>
         </property>
         <property name="maximum">
          <double>360.000000000000000</double>
         </property>
         <property name="value">
          <double>360.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QRadioButton" name="rbDuhamel">
         <property name="contextMenuPolicy">
          <enum>Qt::NoContextMenu</enum>
         </property>
         <property name="text">
          <string>Duhamel, simplified</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup_2</string>
         </attribute>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QRadioButton" name="rbHoerner">
         <property name="contextMenuPolicy">
          <enum>Qt::NoContextMenu</enum>
         </property>
         <property name="text">
          <string>Hoerner, ellipse/cos, Dmax at &gt;</string>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup_2</string>
         </attribute>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QDoubleSpinBox" name="sbDiameter">
         <property name="suffix">
          <string> mm</string>
         </property>
         <property name="maximum">
          <double>100000000000000007629769841091887003294964970946560.000000000000000</double>
         </property>
         <property name="value">
          <double>400.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Length :</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QDoubleSpinBox" name="sbLength">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="suffix">
          <string> mm</string>
         </property>
         <property name="maximum">
          <double>999999999999999926539781176481198923508803215199467887262646419780362305536.000000000000000</double>
         </property>
         <property name="value">
          <double>1000.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="15" column="0">
        <widget class="QRadioButton" name="rbRevolve">
         <property name="statusTip">
          <string/>
         </property>
         <property name="text">
          <string>Revolve :</string>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup</string>
         </attribute>
        </widget>
       </item>
       <item row="8" column="1">
        <widget class="QRadioButton" name="rbNACA">
         <property name="contextMenuPolicy">
          <enum>Qt::NoContextMenu</enum>
         </property>
         <property name="text">
          <string>NACA (4 digits = 00 + D/L %)</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup_2</string>
         </attribute>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Type :</string>
         </property>
        </widget>
       </item>
       <item row="16" column="1">
        <widget class="QDoubleSpinBox" name="sbPadLength">
         <property name="maximum">
          <double>99999999999999.000000000000000</double>
         </property>
         <property name="value">
          <double>0.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="12" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Number of points</string>
         </property>
        </widget>
       </item>
       <item row="12" column="1">
        <widget class="QSpinBox" name="sbNbPoints">
         <property name="minimum">
          <number>10</number>
         </property>
         <property name="maximum">
          <number>1000</number>
         </property>
         <property name="value">
          <number>100</number>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="QDoubleSpinBox" name="sbXMaxRel">
         <property name="maximum">
          <double>1.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.010000000000000</double>
         </property>
         <property name="value">
          <double>0.400000000000000</double>
         </property>
        </widget>
       </item>
       <item row="14" column="0">
        <widget class="QRadioButton" name="rbSketchOnly">
         <property name="text">
          <string>Sketch only</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup</string>
         </attribute>
        </widget>
       </item>
       <item row="16" column="0">
        <widget class="QRadioButton" name="rbPad">
         <property name="statusTip">
          <string/>
         </property>
         <property name="text">
          <string>Pad length :</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup</string>
         </attribute>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QRadioButton" name="rbLyon">
         <property name="contextMenuPolicy">
          <enum>Qt::NoContextMenu</enum>
         </property>
         <property name="text">
          <string>Lyon, model A</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup_2</string>
         </attribute>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Diameter / height :</string>
         </property>
        </widget>
       </item>
       <item row="13" column="0">
        <widget class="QCheckBox" name="ckNewBody">
         <property name="text">
          <string>in a new body</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>rbHoerner</sender>
   <signal>toggled(bool)</signal>
   <receiver>sbXMaxRel</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>186</x>
     <y>95</y>
    </hint>
    <hint type="destinationlabel">
     <x>302</x>
     <y>95</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rbNACA</sender>
   <signal>toggled(bool)</signal>
   <receiver>rbNACA</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>185</x>
     <y>143</y>
    </hint>
    <hint type="destinationlabel">
     <x>185</x>
     <y>143</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
  <buttongroup name="buttonGroup_2"/>
 </buttongroups>
</ui>
